backend/consul/consul.go

```diff
@@ -37,3 +37,8 @@ func (b *Backend) Get(ctx context.Context, key string) ([]byte, error) {
 
 	return kv.Value, nil
 }
+
+// Name returns the name of the backend to compare with the backend tag value.
```

Comment:
You should remove `to compare with the backend tag value`, this package doesn't care.